database/sampleDataMethods.js

```diff
@@ -46,26 +62,19 @@ module.exports.title = title;
 
 /* =================== ISBN generator ===================== */
 
-const isbn = (limit) => {
-  let isbnNum = '';
-  // while isbn length is less than 11;
-  while (isbnNum.length < (limit + 1)) {
-    // add to isbn string;
-    const num = faker.random.number();
-    isbnNum += num.toString();
-  }
-  // splice isbn until 10th char
-  isbnNum = isbnNum.slice(0, limit);
+const isbn = () => {
```

Comment:
perhaps this method, like the others, could be named `getISBN()` (ie, with a `get` prefix) so that they're more in line w/ your decision to name `getRandomInt()` with `get` in its name?